src/dmd/root/region.d

```diff
@@ -69,16 +75,18 @@ struct Region
      */
     void release()
     {
-        void* next;
-        for (auto h = head; h; h = next)
+        version (none)
         {
-            next = *cast(void**)h;
-            memset(h, 0xFC, ChunkSize);
-            .free(h);
+            /* Free the memory
+             */
+            foreach (h; array)
+            {
+                memset(h, 0xFC, ChunkSize); // 0xFC is larger than TOK.max
+                .free(h);
```

Comment:
This `free` is wrong here. Memory should only be stomped, not freed.

Reply:
It's not compiled in at the moment.

Reply:
I know but it will bite you if the code is enabled for debugging. There is no reason to keep it.

Reply:
ok